### PR TITLE
Add Starlette integration to the documentation

### LINKED ISSUE #1284: [DOCS] Can we add Starlette on on Integrations section?
Hi! :wave: 

I'm part of [encode](https://github.com/encode), and we're going to remove the native GraphQL from Starlette, check [here](https://github.com/encode/starlette/pull/1198).

Currently, it's already deprecated, and [we recommend](https://www.starlette.io/graphql/) a set of packages that can be used to work with GraphQL, which strawberry is one of them. And... I'm planning on refactoring the GraphQL docs page on Starlette to point to some different tutorial integrations. 

That being said, I was wondering if we could add an analogous page of the current [FastAPI integration](https://strawberry.rocks/docs/integrations/fastapi) to Starlette. I know it's trivial, but it will be cool anyway. :)

Furthermore, I can create the PR, I just wanted to ask first.

### PR DESCRIPTION
**Pull request by Kludex**
## Description

Add `Starlette` integration to the documentation.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Documentation

## Issues Fixed or Closed by This PR

* Closes #1284 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/integrations/starlette.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Starlette
+---
+
+# Starlette
+
+Strawberry provides support for [Starlette](https://fastapi.tiangolo.com/) with the ASGI integration.
```
Comment (Kludex) [inline docs/integrations/starlette.md:7]:
```suggestion
Strawberry provides support for [Starlette](https://www.starlette.io/) with the ASGI integration.
```

File: docs/integrations/starlette.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from api.schema import schema
+
+graphql_app = GraphQL(schema)
+
+app = Starlette()
+app.add_route("/graphql", graphql_app)
+app.add_websocket_route("/subscriptions", graphql_app)
+```
+
+For more information about Strawberry ASGI refer to [asgi.md](./asgi.md)
```
Comment (patrick91) [inline docs/integrations/starlette.md:24]:
```suggestion
For more information about Strawberry ASGI refer to [the documentation on ASGI](./asgi.md)
```

**Comment by patrick91** [comment]
The ok to preview workflow is broken, but you can see the docs here: https://strawberry.rocks/docs/pr/1287/integrations/starlette :D

**Review by patrick91 — APPROVED** [review]
Looks good to me, I've only requested a minor change 😊

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Comment by Kludex** [comment]
Thanks!

**Pull request merged**